src/NavLink.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5;
+
+use InvalidArgumentException;
+use Stringable;
+use Yiisoft\Html\Html;
+use Yiisoft\Html\Tag\A;
+use Yiisoft\Html\Tag\Li;
+
+/**
+ * Represents a Bootstrap Nav item and link.
+ */
+final class NavLink
+{
+    private const NAV_LINK_ACTIVE_CLASS = 'active';
+    private const NAV_LINK_DISABLED_CLASS = 'disabled';
+    private const NAV_LINK_CLASS = 'nav-link';
+    private const NAV_ITEM_CLASS = 'nav-item';
+    /** @psalm-suppress PropertyNotSetInConstructor */
+    private A|Li $content;
```

Comment:
Whether to use  `<li>` or not should be determined in `Nav` object

Reply:
`NavLink` allows you to create the `item` or `link` that go inside the `nav`, keep in mind that this is used in menu widgets, as well as in the `dropdown`, you choose what you need to use in a simple way.

Reply:
`Nav` can contain either only `LI` or only `A`. So it's should be configured in `Nav` object

Reply:
https://github.com/yiisoft/yii-bootstrap5/blob/3c49fd180d3be51683cbd722632f172596dfa618/src/Nav.php#L210, Here it is controlled, but of course i cannot force its use strictly.

Reply:
done.